src/live/embed.rs

```diff
@@ -454,7 +478,7 @@ impl KargEmbedAreas {
     fn parse(region: &Region) -> Result<String> {
         Ok(String::from_utf8(region.contents.clone())
             .context("invalid UTF-8 in karg area")?
-            .trim_end_matches('#')
+            .trim_end_matches(region.pad.unwrap_or('#'))
             .trim()
```

Comment:
If we're going to support arbitrary end markers, this `trim()` needs to be updated to handle those.